roles/openshift_logging/tasks/install_fluentd.yaml

```diff
@@ -5,6 +5,10 @@
 - set_fact: fluentd_ops_port={{ (openshift_logging_use_ops | bool) | ternary(openshift_logging_es_ops_port, openshift_logging_es_port) }}
   check_mode: no
 
+- set_fact: openshift_logging_fluentd_use_journal={{ "journald" in openshift.docker.options }}
+  check_mode: no
+  when: "openshift_logging_fluentd_use_journal == ''"
```

Comment:
I would prefer to leave fluentd_use_journal unset, or if it must be set, set to `''`, and just let the fluentd pod internally figure out if journal is used or not.